MapboxCoreNavigation/CLLocation.swift

```diff
@@ -3,6 +3,12 @@ import CoreLocation
 
 extension CLLocation {
     
+    var isQualified: Bool {
+        return
+            0...100 ~= horizontalAccuracy &&
```

Comment:
The two modes that we use, `kCLLocationAccuracyBest` and `kCLLocationAccuracyBestForNavigation` are not using wifi, perhaps for a cold start but wifi doesn't support that level of accuracy. However, we should still run the numbers.